.pylintrc

```diff
@@ -1,409 +1,636 @@
-[MASTER]
+[MAIN]
```

Comment:
Regenerated with  `pylint --generate-rcfile`. Old config was generated by old version of pylint. Some options were unsupported and there were missing options.